lectures/matrices/README.md

```diff
@@ -0,0 +1,206 @@
+<h2 align=center>Lecture 03</h2>
+
+<h1 align=center>You're getting old, Matrix</h1>
+
+<h3 align=center>10 Prairial, Year CCXXXI</h3>
+
+***Song of the day***: _[**Cupid (Twin Version)**](https://youtu.be/6uvUTu716rU) by FIFTY FIFTY (피프티피프티) (2023)._
```

Comment:
Could you change this song to [**this one**](https://youtu.be/1w1GUN80KrI?si=2hFvkMeeSW_8weUX)?